module-db/Tables/SMSTemplateTable.cpp

```diff
@@ -21,10 +21,13 @@ bool SMSTemplateTable::create()
 
 bool SMSTemplateTable::add(SMSTemplateTableRow entry)
 {
-    return db->execute("INSERT or ignore INTO templates (text, lastUsageTimestamp) VALUES ('%q', '%q');",
-
-                       entry.text.c_str(),
-                       utils::to_string(entry.lastUsageTimestamp).c_str());
+    auto retQuery       = db->query("SELECT MAX(rowOrder) FROM templates;");
+    auto lastOrderValue = (*retQuery)[0].getUInt32() + 1;
+    return db->execute(
+        "INSERT or ignore INTO templates (text, lastUsageTimestamp, rowOrder) VALUES ('%q', '%q', '%lu');",
```

Comment:
Note: You can also use macros from `module-db/Common/Types.hpp`

Reply:
Apart from that, replace `%lu` with either `u32_` or `PRIu32`

Reply:
Done